translations/translations.json

```diff
@@ -1114,7 +1115,8 @@
       "thanksMessage": "Thank you for your feedback. This will help us to improve our content and the {{appName}} app. We would be happy to receive further feedback.",
       "close": "Close",
       "negativeRating": "This site is not useful",
-      "positiveRating": "This site is useful"
+      "positiveRating": "This site is useful",
+      "contactMailAdress": "Email for possible queries (non-mandatory)"
```

Comment:
```suggestion
```
Just leave it out and we'll let it translate professionally :)

Reply:
ok